### PR TITLE
[shortfin] Fix the f-string for python 3.11

### PR DESCRIPTION
**Pull request by AWoloszyn**

### CODE REVIEW

File: shortfin/python/shortfin_apps/sd/examples/sdxl_flags_gfx1100.txt
Comment (ScottTodd) [inline shortfin/python/shortfin_apps/sd/examples/sdxl_flags_gfx1100.txt:1]: This file is unrelated to the PR. Revert?

File: shortfin/python/shortfin_apps/sd/components/service.py
Comment (ScottTodd) [inline shortfin/python/shortfin_apps/sd/components/service.py:1]: Do we have coverage in CI jobs for this file?

File: shortfin/python/shortfin_apps/sd/examples/sdxl_flags_gfx1100.txt
Comment (AWoloszyn) [reply shortfin/python/shortfin_apps/sd/examples/sdxl_flags_gfx1100.txt:1]: Oops apologies

**Review by stellaraccident — APPROVED** [review]
